v2/logging.go

```diff
@@ -2,44 +2,30 @@ package shuttle
 
 import (
 	"context"
-	"fmt"
-	"os"
-	"time"
+
+	"github.com/sirupsen/logrus"
```

Comment:
I did not take a dependency on a logger on purpose.
what benefit does this provide?
the exported Logger interface already supported levels.
none of the additional levels are used from what I can see?